### ISSUE TITLE
Definition of Standard Deviation/Variance is unclear in parameter table and shrinkage calculation and plots.

### ISSUE THREAD

**Issue by sehamsick** [ggPMXdevelopment/ggPMX#123]
**Issue**
Omega values are not clearly defined in ggPMX for shrinkage calculation.

**For example**
```r
ctr <- theophylline()
ctr %>% pmx_comp_shrink() #will return shrinkage calculated by **variance**
ctr %>% pmx_plot_eta_box() #will return shrinkage calculated by **standard deviation**
```
The function for calculating shrinkage `pmx_comp_shrink()` uses variance as default (changes probably made according to ticket #61). However, the function can also use standard deviation  `pmx_comp_shrink(fun = "sd")`. This function relies on `ctr %>% get_data("estimates")` or `ctr %>% get_data("omega")`, in order to get the omega values for shrinkage calculation.

Even though `pmx_comp_shrink()` has variance as default, ggPMX uses standard deviation for creating the controller and the plots e.g. `ctr %>% pmx_plot_eta_box()` 

**Basically there are two issues:**
 -  It is not clear if the omega values in the parameter table, which can be retrieved by `ctr %>% get_data("estimates")`, is the variance or the standard deviation.
-  The default (either using standard deviation or variance) for  `pmx_comp_shrink()` is not the same as the default for the `ctr` generation or the plots e.g. `pmx_plot_eta_box()`

**Comment by baltcir1** [comment]
Solving this would also solve https://github.com/ggPMXdevelopment/ggPMX/issues/53
